.github/workflows/lint.yml

```diff
@@ -0,0 +1,18 @@
+name: Lint
+
+on: [push, pull_request, workflow_dispatch]
+
+permissions:
+  contents: read
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
```

Comment:
Is it worth it pinning this to a version? (I guess chances are low this will break when GitHub bump the Ubuntu runner, so we can probably leave it at `ubuntu-latest`.)

Reply:
`ubuntu-latest` should be fine here, I can't recall it ever breaking on a version change for pre-commit on other projects.